changelog.txt

```diff
@@ -1,3 +1,8 @@
+2.7.1
```

Comment:
It's possible that you are using the old "master". Please try to delete your master branch and fetch it from the remote again. Then reopen this PR.

Reply:
@bgavrilMS  it was the PR itself. It defaults to master as the branch to merge into. I changed the target to archive/release2x and everything is good

Reply:
Ah ok, got it. So we want to patch MSAL 2.x with this fix?

Reply:
Oh sorry, I got confused, I see what you are doing. LGTM.